pkgs/tools/misc/remind/default.nix

```diff
@@ -21,6 +21,14 @@ in stdenv.mkDerivation {
   nativeBuildInputs = optional tkremind makeWrapper;
   propagatedBuildInputs = tclLibraries;
 
+  patchPhase = ''
```

Comment:
Actually this should be put in postPatch below. Otherwise you end up overriding any patch files that were provided.

Reply:
Fixed in latest push (and tested that it still works as expected).